lib/composer

```diff
@@ -68,7 +68,7 @@ function install_composer_deps() {
             elif [ "$ext" = "tidy" ] ; then
               fetch_package "libtidy-${tidy_version}" "/app/vendor/libtidy" | indent
             fi
-            fetch_package "ext/$(php_api_version)/php-${ext}" "/app/vendor/php" 2>/dev/null || true | indent
+            fetch_package "ext/$(php_api_version)/php-${ext}" "/app/vendor/php" 2>/dev/null | indent || true
```

Comment:
Why this change? And actually, why all the changes except for the one on line 60 of this file?

Reply:
This change was for testing, I go back this change, it was not necessary to update this.

For the other changes (indent() function), I wanted the functions to be the same everywhere, since the behaviour of these functions is the same. (It add principally the comments)